GitCommands/Git/GitModule.cs

```diff
@@ -1642,14 +1642,21 @@ public string ApplyPatch(string dir, ArgumentString amCommand)
 
         #region Stage/Unstage
 
-        public string AssumeUnchangedFiles(IReadOnlyList<GitItemStatus> files, bool assumeUnchanged, out bool wereErrors)
+        /// <summary>
+        /// Set/unset assumeUnchanged
+        /// </summary>
+        /// <param name="files">Files to change</param>
+        /// <param name="assumeUnchanged">Set/unset assumeUnchanged</param>
+        /// <param name="alloutput">stdout and stderr output</param>
+        /// <returns>true if no errors occurred</returns>
```

Comment:
The docs should be in sentence case and end with a full stop. We should also strive to use xml-doc tags, where appropriate.
```suggestion
        /// <summary>
        /// Set/unset whether given items are assumed unchanged.
        /// </summary>
        /// <param name="files">The files to set the status for.</param>
        /// <param name="assumeUnchanged">The status value.</param>
        /// <param name="allOutput">stdout and stderr output.</param>
        /// <returns><see langword="true"/> if no errors occurred; otherwise, <see langword="false"/>.</returns>
```

Ditto below.

Reply:
Updated.
(Not too happy with the xml-doc, the primary intention is to mark that AllOutput is deliberate, not being parsed.)